web-portal/backend/src/guards/auth.guard.ts

```diff
@@ -0,0 +1,13 @@
+import { CanActivate, ExecutionContext, Injectable } from '@nestjs/common';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class AuthGuard implements CanActivate {
+  canActivate(
+    context: ExecutionContext,
+  ): boolean | Promise<boolean> | Observable<boolean> {
+    console.log('Inside AuthGuard');
+    const request = context.switchToHttp().getRequest();
+    return true; // @note: temp  returning true for testing
```

Comment:
Is this for local testing or did you mean for this to make it to `develop`?

Reply:
Didn't mean to but didn't revert; working off this!